lib/t/matches.ex

```diff
@@ -428,26 +428,37 @@ defmodule T.Matches do
 
     broadcast_for_user(mate, {__MODULE__, [:timeslot, :accepted], timeslot})
 
-    accepted_push =
-      DispatchJob.new(%{
-        "type" => "timeslot_accepted",
-        "match_id" => match_id,
-        "receiver_id" => mate
-      })
+    if DateTime.utc_now() > slot do
```

Comment:
use `reference` instead of `DateTime.utc_now()`, easier to test, stays consistent within the function

Reply:
also don't use comparison operators on datetimes, it doesn't work

Reply:
use `DateTime.compare`

Reply:
please add a test

Reply:
@ruslandoga how do I test insertion of push notification in oban_jobs?

Reply:
https://github.com/getsince/test3/blob/346a4866b9112deed13437f7548a7c8fb907c442/test/t/matches/timeslots_test.exs#L271